Tests/Unit/NetworkServer/LoRaDeviceTest.cs

```diff
@@ -1057,5 +867,27 @@ public void BeginDeviceClientConnectionActivity_Delegates_To_Connection_Manager_
             connectionManagerMock.Verify(x => x.BeginDeviceClientConnectionActivity(target),
                                          Times.Exactly(timeoutSeconds > 0 ? 1 : 0));
         }
+
+        private static Twin CreateTwin(Dictionary<string, object> desired = null, Dictionary<string, object> reported = null)
```

Comment:
Wouldn't it make sense to use the `LoRaTwinProperties` records here too?

Reply:
I don't think so, but maybe we can do better. The `LoRaTwinProperties` simplify the setup of *correct* test twins for us. Some tests in this class have assertions on the data contracts - sometimes by using different representation of properties (e.g. an `int` as a number and as a string once, while expecting that our code can handle both). If we were to use the `LoRaTwinProperties` in all cases, we would not get the same coverage. This is why I kept a `private` helper method to create a twin from these dictionaries containing arbitrary values.

Reply:
I see. I looked at the only 3 usages and nothing struck me as testing the various representation, which is why I asked the question. Perhaps I missed a case or two?

Reply:
You are right, I mixed things up with other occurrences where I am setting a property directly on the twin. Addressed with c03d6d5b38588ca4dc0b6d8f7da194698e229285